pkg/lib/cockpit-components-plot.jsx

```diff
@@ -411,10 +412,13 @@ export const SvgPlot = ({ title, config, plot_state, plot_id, onHover, className
 
         return (
             <svg width={w} height={h}
+                 ariaLabel={title}
```

Comment:
This needs to be `aria-label`.

Reply:
Ah, right. I forgot. React is inconsistent here.